docs/howtoguides/trusty_legacy_support.rst

```diff
@@ -0,0 +1,98 @@
+How to enable esm-infra-legacy on Trusty
+*****************************************
+
+The ``esm-infra-legacy`` service can be enabled to grant additional two years of 
+extend security coverage. This service is only available on Trusty, since the release has reached
+its end of life support for esm-infra.
+
+.. Make sure Pro is up to date
+.. include:: ./enable-disable/update-pro.txt
+
+Check the status of the services
+================================
+
+After you have :ref:`attached your subscription <get_token_and_attach>` and
+updated the ``ubuntu-advantage-tools`` package, you can check which services
+are enabled by running the following command:
+
+.. code-block:: bash
+
+    ua status
+
+This will show you which services are enabled or disabled on your machine
+(output truncated for brevity):
+
+.. code-block:: text
+
+    SERVICE          ENTITLED  STATUS    DESCRIPTION
+    esm-infra        yes       enabled   Expanded Security Maintenance for Infrastructure
+    esm-infra-legacy yes       disabled  Expanded Security Maintenance for Infrastructure on Legacy Instances
+    livepatch        yes       enabled   Canonical Livepatch service
+
+If you are entitled to the service, you can simply enable it by running:
+
+
+.. code-block:: bash
+
+    sudo ua enable esm-infra-legacy
+
+
+What to do if esm-infra-legacy is not entitled
+===============================================
+
+If status is showing that you are not entitled to the service, you
+will need to buy access to the service through the `Pro webpage`_.
+
+Once you have bought the support, you will need to run the following command
+on your machine:
+
+.. code-block:: bash
+
+    sudo ua refresh
+
+
+This command will refresh the contract definitions on your machine. After that, you
+can confirm that the service is now entitled by running:
+
+
+.. code-block:: bash
+
+    ua status
+
+If you are now entitled to the service, you can enable it with:
+
+.. code-block:: bash
+
+    sudo ua enable esm-infra-legacy
+
+
+Trusty caveats
+=========================================
+
+There are some well known caveats for the Trusty version of `ubuntu-advantage-tools`:
+
+* **Disabling esm-infra also disables esm-infra-legacy**: Even though this is happening, users
+  can disable esm-infra-legacy without consequences for esm-infra
+* **do-release-upgrade fails if esm-infra enabled and packages installed from service**: The
+  do-release-upgrade operation will fail here, as there will be an APT dependency chain issue
+  when performing the operation.
+
+Please note that all of these problems won't happen on Xenial forwards. The client is still fully
+supported on those later releases and these issues have already been fixed for them.
+
+Why 14.04 (Trusty) is no longer supported
+=========================================
```

Comment:
I think this can become a subheading under "trusty caveats" - I think that means the underline changes but not sure